src/server/storage/storageAPI.js

```diff
@@ -39,7 +39,8 @@ const setup = (app: Router, storage: StorageAPI) => {
       )
         throw new Error('User email or mobile not verified!')
 
-      const { email, mobile, ...bodyUser } = body.user
+      const bodyUser = body.user
```

Comment:
this is not good since we can receive anything here from the user.
read "savedEmail","savedMobile" from the existing user record which are the verified details

Reply:
user.otp  has this data